documentation/src/main/asciidoc/topics/code_examples/ManualSerializationContextInitializer.java

```diff
@@ -0,0 +1,28 @@
+import org.infinispan.protostream.FileDescriptorSource;
+import org.infinispan.protostream.SerializationContext;
+import org.infinispan.protostream.SerializationContextInitializer;
+...
+
+public class ManualSerializationContextInitializer implements SerializationContextInitializer {
+   @Override
+   public String getProtoFileName() {
+      return "/library.proto";
```

Comment:
is normal to have a / ?

Reply:
Yes, this is required as we're loading it from the Jar's resources.

Reply:
I have to admit this is a little confusing.. The generated one doesn't have this slash, but that should be in the Jar too..

Reply:
O my bad ... the "/" is required for `getProtoFile` not `getProtoFileName` :see_no_evil: